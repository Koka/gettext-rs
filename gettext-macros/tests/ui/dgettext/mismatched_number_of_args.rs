use gettext_macros::*;

fn main() {
    dgettext!("domainname", "{}");
    dgettext!("domainname", "{}", 1, 2);
    dgettext!("domainname", "{}{}");
    dgettext!("domainname", "{}{}", 1);
    dgettext!("domainname", "{}{}{}", 1, 2);
}
