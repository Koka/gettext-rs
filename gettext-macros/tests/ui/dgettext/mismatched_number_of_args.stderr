error: 1 positional argument in format string, but no arguments were given
 --> tests/ui/dgettext/mismatched_number_of_args.rs:4:29
  |
4 |     dgettext!("domainname", "{}");
  |                             ^^^^

error: 1 positional argument in format string, but there are 2 arguments
 --> tests/ui/dgettext/mismatched_number_of_args.rs:5:29
  |
5 |     dgettext!("domainname", "{}", 1, 2);
  |                             ^^^^

error: 2 positional arguments in format string, but no arguments were given
 --> tests/ui/dgettext/mismatched_number_of_args.rs:6:29
  |
6 |     dgettext!("domainname", "{}{}");
  |                             ^^^^^^

error: 2 positional arguments in format string, but there is 1 argument
 --> tests/ui/dgettext/mismatched_number_of_args.rs:7:29
  |
7 |     dgettext!("domainname", "{}{}", 1);
  |                             ^^^^^^

error: 3 positional arguments in format string, but there are 2 arguments
 --> tests/ui/dgettext/mismatched_number_of_args.rs:8:29
  |
8 |     dgettext!("domainname", "{}{}{}", 1, 2);
  |                             ^^^^^^^^
